inc/rack.class.php

```diff
@@ -891,7 +891,19 @@ classes: 'qtip-shadow qtip-bootstrap rack_tipcontent'
    }
 
    function prepareInputForAdd($input) {
-      return $this->prepareInput($input);
+      if ($this->prepareInput($input)) {
+         if (isset($input["id"]) && ($input["id"] > 0)) {
+            $input["_oldID"] = $input["id"];
+         }
+         if (isset($input["size"]) && ($input["size"] == '')) {
```

Comment:
I do not understand this one; there is no size field in racks

Reply:
So I guess this should just be removed. Rest looks good (and fix the original issue)